popde/adaptive_kde.py

```diff
@@ -117,6 +118,98 @@ def set_adaptive_parameter(self, new_alpha, new_global_bw):
         self.set_alpha(new_alpha)
 
 
+class KDEOptimization(AdaptiveBwKDE):
+    """
+    Given grid of bandwidth and alpha values 
+    find optimized values using cross 
+    valitation with figure of merit
+    based on loglikelihood criteria 
+    """
+    def __init__(self, data, weights, bandwidth_options, alpha_options , input_transf=None, stdize=False, rescale=None, backend='KDEpy', bandwidth=0.5, alpha=0.0, dim_names=None, do_fit=True):
```

Comment:
I think weights should be a kwarg with default None. 
Also why is bandwidth being given a default value 0.5?  I guess we don't need any default value as this is ignored but 0.5 seems a weird choice ..

Reply:
added as kwarg

Reply:
bw value is 1 by default